rosbag2_cpp/src/rosbag2_cpp/reindexer.cpp

```diff
@@ -86,8 +86,8 @@ bool Reindexer::compare_relative_file(
     throw std::runtime_error(error_text.c_str());
   }
 
-  u_int32_t first_db_num = std::stoul(first_match.str(1), nullptr, 10);
-  u_int32_t second_db_num = std::stoul(second_match.str(1), nullptr, 10);
+  auto first_db_num = std::stoul(first_match.str(1), nullptr, 10);
```

Comment:
I think that auto is fine here - but as a note, the portable C++ type is called `uint32_t` (only one underscore)

Reply:
Ah. That explains a lot.

I _THOUGHT_ that type looked odd, but my editor seemed to like it...